### PR TITLE
mat-671-use-di-and-convert-to-kotlin

### PR DESCRIPTION
**Pull request by abhriyaroy**

### CODE REVIEW

File: src/test/java/network/matic/dagger/test/TestDaggerInvocationHelper.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package network.matic.dagger.test
```
Comment (jdkanani) [inline src/test/java/network/matic/dagger/test/TestDaggerInvocationHelper.kt]: Is this file needed?

**Review by jdkanani — CHANGES_REQUESTED** [review]

File: src/test/java/network/matic/dagger/test/TestDaggerInvocationHelper.kt (hunk 1 above)
Comment (abhriyaroy) [reply src/test/java/network/matic/dagger/test/TestDaggerInvocationHelper.kt]: Yes will be adding test cases in next pr. Can delete file in current one.

**Review by jdkanani — APPROVED** [review]
